src/main/java/net/dries007/tfc/world/surface/builder/SaltFlatsSurfaceBuilder.java

```diff
@@ -0,0 +1,26 @@
+package net.dries007.tfc.world.surface.builder;
+
+import net.dries007.tfc.world.noise.Noise2D;
+import net.dries007.tfc.world.noise.OpenSimplex2D;
+import net.dries007.tfc.world.surface.SurfaceBuilderContext;
+import net.dries007.tfc.world.surface.SurfaceStates;
+
+public class SaltFlatsSurfaceBuilder implements SurfaceBuilder
```

Comment:
This and the previous surface builder could be merged, and take the surface states as constructor parameters, reduces some duplication.